eofconvention.fsx

```diff
@@ -0,0 +1,29 @@
+open System.IO
+
+let EOFIsEOL(path: FileInfo) = 
+    try
+        use sr = new StreamReader (path.FullName)
+        let filetext = sr.ReadToEnd()
+        (filetext |> Seq.last = '\n')
+        
+    with
+        | :? System.ArgumentException -> printfn "%A" path; true
+        
+
+let EOFIsNotEOL(path: FileInfo) = not (EOFIsEOL path)
+
+let NotPrefix (value: string) (str: string) =
+    not (str.StartsWith(value))
+
+
+let invalidFiles = 
+    Directory.GetFiles(".", "*.*", SearchOption.AllDirectories) 
+    |> Array.filter (NotPrefix "./node_modules")
+    |> Array.filter (NotPrefix "./.git")
+    |> Array.map (fun x -> FileInfo(x))
```

Comment:
@realmarv nits about this line:

- No need for parentheses when calling FileInfo
- Please use a longer name for variable x (otherwise when we adopt FSharpLint in this repo, it will fail here)

Reply:
Done